modules/test/playwright/tests/commerce/commerce-order-content-web/placedOrder.spec.ts

```diff
@@ -265,3 +272,164 @@ test('LPD-26643 Reorder from placed orders details page', async ({
+	});
+
+	const account1 = await apiHelpers.headlessAdminUser.postAccount({
+		name: getRandomString(),
+		type: 'business',
+	});
+
+	apiHelpers.data.push({id: account1.id, type: 'account'});
+
+	await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account1.id,
+		[userAccount.emailAddress]
+	);
+
+	const rolesResponse1 = await apiHelpers.headlessAdminUser.getAccountRoles(
+		account1.id
+	);
+
+	const accountRoleBuyer1 = rolesResponse1?.items?.filter((role) => {
+		return role.name === 'Buyer';
+	});
+
+	await apiHelpers.headlessAdminUser.assignAccountRoles(
+		account1.externalReferenceCode,
+		accountRoleBuyer1[0].id,
+		userAccount.emailAddress
+	);
+
+	const account2 = await apiHelpers.headlessAdminUser.postAccount({
+		name: getRandomString(),
+		type: 'business',
+	});
+
+	apiHelpers.data.push({id: account2.id, type: 'account'});
+
+	apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account2.id,
+		[userAccount.emailAddress]
+	);
+
+	await applicationsMenuPage.goToSite(site.name);
+
+	await commerceLayoutsPage.goToPages(false);
+	await commerceLayoutsPage.createWidgetPage('Placed Orders Page');
+
+	await page.goto(`/web/${site.name}`);
+
+	await placedOrdersPage.addPlacedOrdersWidget();
+
+	const productSkus = await apiHelpers.headlessCommerceAdminCatalog
+		.getProduct(product.productId)
+		.then((product) => {
+			return product.skus;
+		});
+
+	const sku = productSkus[0];
+
+	const phoneNumber = '12345';
+
+	const address1 = await apiHelpers.headlessCommerceAdminAccount.postAddress(
+		account1.id,
+		{phoneNumber, regionISOCode: 'AL'}
+	);
+
+	await apiHelpers.headlessCommerceAdminOrder.postOrder({
+		accountId: account1.id,
+		billingAddressId: address1.id,
+		channelId: channel.id,
+		orderItems: [
+			{
+				decimalQuantity: 10,
+				quantity: 2,
+				skuId: sku.id,
+			},
+		],
+		orderStatus: '0',
+		paymentMethod: 'paypal',
+		paymentStatus: '0',
+		shippingAddressId: address1.id,
+	});
+
+	const address2 = await apiHelpers.headlessCommerceAdminAccount.postAddress(
+		account2.id,
+		{phoneNumber, regionISOCode: 'AL'}
+	);
+
+	await apiHelpers.headlessCommerceAdminOrder.postOrder({
+		accountId: account2.id,
+		billingAddressId: address2.id,
+		channelId: channel.id,
+		orderItems: [
+			{
+				decimalQuantity: 10,
+				quantity: 2,
+				skuId: sku.id,
+			},
+		],
+		orderStatus: '0',
+		paymentMethod: 'paypal',
+		paymentStatus: '0',
+		shippingAddressId: address2.id,
+	});
+
+	await performLogout(page);
+	await performLogin(page, userAccount.alternateName);
+
+	try {
+		await page.goto(`/web/${site.name}`);
+
+		await placedOrdersPage.searchInput.fill(account2.name);
+		await placedOrdersPage.searchButton.click();
+
+		await expect(
+			placedOrdersPage.orderAccountName(account1.name)
+		).toHaveCount(0);
+		await expect(
+			placedOrdersPage.orderAccountName(account2.name)
+		).toHaveCount(1);
+	}
+	finally {
+		await performLogout(page);
+		await performLogin(page, 'test');
+	}
```

Comment:
I think this is not needed anymore because the teardown now is done using always admin. Can you try locally to verify if it deletes everything also without this?

Reply:
It works without it.